app/models/domain.rb

```diff
@@ -0,0 +1,3 @@
+class Domain < ApplicationRecord
+  has_many :stories, dependent: :destroy
```

Comment:
Was this prompted by a rubocop rule or something?

I'm unsettled by having an accessory models like Domain that's created-in-time and not very important with a `dependent: :destroy` pointing at Story. If we deleted one of these records to recreate it, oops, we'd delete a bunch of stories + comments, our core features. It feels like a gun waiting to go off.

Any ideas for making this less of a landmine? Can we remove the `dependent: :destroy`, override the `destroy` method on the class, anything else? I'd just rather `stories` have some broken FKs than have to restore from database backups.

Reply:
We can remove the `:destroy` and suppress the cop inline. That feels like the cleanest situation.